build_tools/ci/run_matmul_test.sh

```diff
@@ -411,8 +395,7 @@ function run_matmul_test() {
     if [ -f "${OUTPUT_DIR}/mm.o" ]; then
       echo "File 'mm.o' already exists in ${OUTPUT_DIR}."
     else
-      SRC_DIR="${mlir_aie_install_path}/aie_kernels/mm.o"
-      ln -s ${SRC_DIR} ${OUTPUT_DIR}/mm.o
+      wget $MM_KERNEL_URL -O  ${OUTPUT_DIR}/mm.o
```

Comment:
do we need this? looks like we will definitely have the object where we need it so we can remove any reference to it in the CI script?

Reply:
are we? I don't see anywhere in aie2xclbin where we handle ukernels?

Reply:
my bad, I thought this was m[e_basic].o stuff, I get it now, thanks.